.gitignore

```diff
@@ -1,35 +1 @@
-*.py[cod]
-
-# C extensions
-*.so
-
-# Packages
-*.egg
-*.egg-info
-dist
-build
-eggs
-parts
-bin
-var
-sdist
-develop-eggs
-.installed.cfg
-lib
-lib64
-
-# Installer logs
-pip-log.txt
-
-# Unit test / coverage reports
-.coverage
-.tox
-nosetests.xml
-
-# Translations
-*.mo
-
-# Mr Developer
-.mr.developer.cfg
-.project
-.pydevproject
+.idea
```

Comment:
Why has the whole gitignore been removed?

Reply:
Not! I will not return the .gitignore file back! Just out of principle! You have a complete shit there!

Reply:
I don’t care if you accept my pull register or not - your module is now mine mine! And I decide how it will be and how it will work - https://github.com/HeaTTheatR/FilechooserThumbview

Reply:
Dude WTF? :joy:

Reply:
@Depau Hit your head on the table!